src/jobs/PkgEvalJob.jl

```diff
@@ -788,6 +792,9 @@ function report(job::PkgEvalJob, results)
                             font-family: sans-serif;
                             max-width: 65rem;
                         }
+                        table td + td + td + td + td {
+                            font-family: monospace;
+                        }
```

Comment:
This will only work for the 5-column tables. For the 3-column tables (no tests / could not be installed / blacklisted) there will be no effect. It would be possible to give the two table widths different classes; we could even coopt the color map to do that, though it might lead to strange style artifacts later—regexs are just powerful enough to be dangerous.